cnf-tests/testsuites/e2esuite/rps/rps.go

```diff
@@ -0,0 +1,268 @@
+package rps
```

Comment:
maybe the name and folder should be pao or performance_profile?

Reply:
Changed to pao. Thanks!